### ISSUE TITLE
Error when launching sclang, after updating readline

### ISSUE THREAD

**Issue by ritchse** [supercollider/supercollider#5593]
## Environment

* SuperCollider version: 3.11.2
* Operating system: Lubuntu
* Other details (Qt version, audio driver, etc.): Readline version: 8.1

## Steps to reproduce

I installed the latest readline version from the [official website](https://ftp.gnu.org/gnu/readline/).
Then, well, this happened: 

```bash
ritchse@ritchse-hp-dv4:~$ sclang -v
sclang: symbol lookup error: /usr/local/lib/libreadline.so.8: undefined symbol: UP
```
And on the IDE:
```
Interpreter has quit. [Exit code: 127]
```

Any info you need I'll share, sorry if there's something obvious missing.

**Comment by dyfer** [comment]
Hello @ritchse 
Have you installed SC using a your system package manager or did you compile it yourself?
IIUC if using SC from a package manager, you are expected to use readline version from the package manager, too. 
If you compiled SC yourself, you probably need to recompile it against the updated version of readline.

**Comment by dyfer** [comment]
I haven't used the linked script, so I can't comment on that.

When SC configures (before compiling), it should inform you whether readline was found or not. 
As for the linked issue, it seems that the proper solution is to have `libreadline-devel` package installed. That's probably preferred to installing a library from outside of the package manager (remember to uninstall the one you've installed from the official website first).
You'll need to recompile SC anyway to get it to use the new library, either the one installed through a package manager or the one you manually installed.

**Comment by ritchse** [comment]
Okay, I updated SC, uninstalled readline 8.1 and installed libreadline-dev. Everything working now, except for the interpreter when loading a file. But that's for another issue maybe. I'm using a workaround now tho, I just launch the interpreter with no file and then use this function to interpret a file, which I've added to my startup.scd.

```supercollider
(
~runFile = { |dir|
        thisProcess.interpreter.interpret(File.readAllString(dir));
};
)
```

Given the mentioned reasons I close the issue. Thanks for the help @dyfer ! I really appreciate it.

**Comment by alln4tural** [comment]
quite possibly I'm missing something, but it looks to me like you just want to .load the filehttp://doc.sccode.org/Classes/String.html#-loadCheers,eddihttps://alln4tural.bandcamp.comOn 10/10/2021, 21:51 Renzo Torr- ***@***.***> wrote:

   
  Okay, I updated SC, uninstalled readline 8.1 and installed libreadline-dev. Everything working now, except for the interpreter when loading a file. But that's for another issue maybe. I'm using a workaround now tho, I just launch the interpreter with no file and then use this function to interpret a file, which I've added to my startup.scd. 
  
   (
~runFile = { |dir|
        thisProcess.interpreter.interpret(File.readAllString(dir));
};
)
   
  Given the mentioned reasons I close the issue. Thanks for the help @dyfer ! I really appreciate it.
